### PR TITLE
 CVE-2013-0889

### PR DESCRIPTION
**Pull request by thatapplefreak**

### CODE REVIEW

**Comment by xdmds** [comment]
The assignment requires 2 CVEs?

**Comment by thatapplefreak** [comment]
@xdmds Please see https://github.com/andymeneely/chromium-vulnerabilities/pull/86 I worked on separate branches so two requests were made.

File: cves/CVE-2013-0889.yml [hunk 1]
```diff
@@ -32,7 +35,9 @@ fixes_vcc_instructions: |
 fixes:
 - :commit: 1538367452b549d929aabb13d54c85ab99f65cd3
   :note: ''
-vccs: []
+vccs:
+- :commit: 3d95e54652070dacfdb9188c3f6f48111bcc007e
```
Comment (JurelCastillo) [inline cves/CVE-2013-0889.yml]: Nice! you found at least 1

File: cves/CVE-2013-0889.yml [hunk 2]
```diff
@@ -93,8 +101,10 @@ interesting_commits:
     interesting in light of the lessons learned from this vulnerability. Any
     emerging themes?
   commits:
-  - commit: 
-    note: 
+  - commit: cda790613b518dee5a3ea4ce7d83989c1acd166a
+    note: |
+      This change was made right around the vulenrability, a constant used for
```
Comment (JurelCastillo) [inline cves/CVE-2013-0889.yml]: vulnerability*

File: cves/CVE-2013-0889.yml [hunk 3]
```diff
@@ -139,8 +147,11 @@ lessons:
     applies: 
     note: 
   distrust_input:
-    applies: 
-    note: 
+    applies: true
```
Comment (JurelCastillo) [inline cves/CVE-2013-0889.yml]: I have to agree with this

**Comment by JurelCastillo** [comment]
Good job overall

File: cves/CVE-2013-0889.yml [hunk 4]
```diff
@@ -15,7 +15,10 @@ description_instructions: |
   that outsiders to Chromium would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description: 
+description: |
```
Comment (dxf7606) [inline cves/CVE-2013-0889.yml]: 2 upvotes

**Pull request merged**